src/app/ui/ui-select/ui-select.component.scss

```diff
@@ -16,7 +16,8 @@
       box-sizing: border-box;
       min-width: 140px;
       padding:0;
-      max-height: 190px;
```

Comment:
This is mostly for mobile, to make sure the content of the select dropdowns don't go off of the screen.  Happens mostly for year selector.  Could you remove the max height in a media query instead?

Reply:
Makes sense! I'll update that